test/integration/PartnerRegistrar.spec.ts

```diff
@@ -309,18 +309,6 @@ describe('New Domain Registration', () => {
       )
     ).wait();
 
-    await time.increase(1);
-
-    const canReveal = await partnerProxyAsNameOwner.canReveal(commitment);
-
-    expect(canReveal).to.be.true;
-
-    await time.increase(1);
-
-    const canReveal = await partnerProxyAsNameOwner.canReveal(commitment);
-
-    expect(canReveal).to.be.true;
-
```

Comment:
why removed? 👀

Reply:
That's a mistake, I was supposed to delete the duplicate and then correct the mistake on line 320. I've fixed it now, thanks for spotting it